tests/suites/test_suite_psa_crypto.data

```diff
@@ -5185,6 +5185,55 @@ PSA key derivation: TLS 1.2 PSK-to-MS, SHA-384, 0+48
 depends_on:PSA_WANT_ALG_SHA_384:PSA_WANT_ALG_TLS12_PSK_TO_MS
 derive_output:PSA_ALG_TLS12_PSK_TO_MS(PSA_ALG_SHA_384):PSA_KEY_DERIVATION_INPUT_SEED:"5bed47716a11a49a6268a8350b085929116ad9ccc8181f09a05b07a7741576d65bed47718dfd82f2d3f57544afe52decae6819b970dc716ada72ae0dd3072e9a":PSA_KEY_DERIVATION_INPUT_SECRET:"01020304":PSA_KEY_DERIVATION_INPUT_LABEL:"6d617374657220736563726574":0:"":48:"f5a61fbdd2ec415762abb8042a6c16645a53d2edb6dec8c8":"5ca71689301f9f4d875128c87608b75250b20a9550e4fe18"
 
+# sl = salt length in bytes
+PSA key derivation: TLS 1.2 Mix-PSK-to-MS, SHA-256, 48+0, sl 48
+depends_on:PSA_WANT_ALG_SHA_256:PSA_WANT_ALG_TLS12_PSK_TO_MS
+derive_output:PSA_ALG_TLS12_PSK_TO_MS(PSA_ALG_SHA_256):PSA_KEY_DERIVATION_INPUT_SEED:"5bc0b19b4a8b24b07afe7ec65c471e94a7d518fcef06c3574315255c52afe21b5bc0b19b872b9b26508458f03603744d575f463a11ae7f1b090c012606fd3e9f":PSA_KEY_DERIVATION_INPUT_SALT:"ce2fa604b6a3e08fc42eda74ab647adace1168b199ed178dbaae12521d68271d7df56eb56c55878034cf01bd887ba4d7":PSA_KEY_DERIVATION_INPUT_SECRET:"01020304":PSA_KEY_DERIVATION_INPUT_LABEL:"6d617374657220736563726574":48:"168fecea35190f9df34c042f24ecaa5e7825337f2cd82719464df5462f16aae84cb38a65c0d612ca9273f998ad32c05b":""
```

Comment:
Where does the data come from? I assume for logs of a successful RSA-PSK handshake (and DHE-PSK) below? It would be good to have a comment about it (or at least mention it in the commit message). Also, we probably should have some from data from an ECDHE-PSK exchange (which gives a smaller salt with the default curve).

Reply:
In the PR description I added information how test vectors were generated.

Reply:
Committing the script may be more work than we care to do. This requires making it pass `check-python-files`, which requires adding its dependencies to `ci.requirements.txt`.

I also don't necessarily prefer having a comment inside the file to indicate how the data was generated. It can be difficult to convey precisely which test cases the comment applies to, and there's a fairly high risk that new data will be added or that the data will be modified, causing the scope described in the comment to become wrong.

So my preference is to have instructions for generating the test data in the commit message.

Reply:
Sure I can commit this script and provide details in the commit. Oh and still need to add change log entry.